tun/tun_linux.go

```diff
@@ -380,6 +380,9 @@ func (tun *NativeTun) Close() error {
 func CreateTUN(name string, mtu int) (Device, error) {
 	nfd, err := unix.Open(cloneDevicePath, os.O_RDWR, 0)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("can't create TUN device; %s does not exist", cloneDevicePath)
```

Comment:
Maybe include some words about the tuntap kernel module, so that folks have something to search for to resolve?

Reply:
Extra words added to improve searching, I hope.